index.js

```diff
@@ -376,5 +379,25 @@ const testCommand = (arguments, receivedMessage) => {
   );
 };
 
+const verifyCommand = (arguments, receivedMessage) => {
+  if (arguments.length > 0){
+    receivedMessage.channel.send(
+      "Invalid arguments given."
+    );
+  }
+  else{
+    if (receivedMessage.member.roles.cache.has("698842106613202944") || receivedMessage.member.roles.cache.has("698829800965668884") || receivedMessage.member.roles.cache.has("699065951123013652")){
+      receivedMessage.guild.roles.cache.get("698841628856811601").members.forEach(m => {
+          return m.send("Please verify yourself. For help, dm the OIC or any of the school leads");
```

Comment:
Can you use the same message that is used in the original confirm message here instead? ie how they shud verify. 

Since most of the people would not be able to find the original message.